ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py

```diff
@@ -1136,3 +1136,46 @@ def test_paddle_subtract(
         frontend=frontend,
         on_device=on_device,
     )
+
+
+# amin
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="amin",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=st.one_of(helpers.get_dtypes("float", "int")),
```

Comment:
you can put `numeric` in the `get_dtypes` method and add `complex` as unsupported dtypes if they are not supported